### PR TITLE
(DOCSP-26939) [Section] Monitor App Activity

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-26939) [Section] Monitor App Activity

### Staged Changes

- [Monitor App Activity](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-activity/activity/)

### CODE REVIEW

File: source/activity.txt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+App Services saves only the first 25 log lines for a given log entry.
+
+Log Retention
+~~~~~~~~~~~~~
+
+App Services retains logs for {+log-retention-time+}, after which they
+are deleted.
+
+If you require logs older than {+log-retention-time+}, you can
+automatically :ref:`forward logs <forward-logs>` to another service. You
```
Comment (cbullinger) [inline source/activity.txt:114]: Not seeing this page in the left toc after clicking link

File: source/activity/view-logs.txt [hunk 2]
```diff
@@ -1,26 +1,141 @@
...
 
-Overview
---------
+App Services UI
+---------------
 
-You can request an application's logs programmatically by accessing the
-logging endpoints of the :doc:`Atlas App Services Admin API </admin/api/v3>`.
+You can view, filter, search, and download an application's logs in the
+App Services UI. To navigate to the logs screen, click :guilabel:`Logs`
```
Comment (cbullinger) [inline source/activity/view-logs.txt:17]: "To navigate to the logs screen..." we don't format or capitalize screen names at all? Just `the logs screen` reads oddly to me.

File: source/activity/view-logs.txt [hunk 3]
```diff
@@ -1,26 +1,141 @@
...
+
+App Services API
+----------------
+
+You can access an application's logs over HTTPS by calling the
+:admin-api-endpoint:`Admin API Logs endpoints <tag/logs>`.
+
+To use the Admin API you'll need your Project ID, App ID, and
+authentication credentials. To learn how to find these, see
+:admin-api-endpoint`Project & Application IDs
```
Comment (cbullinger) [inline source/activity/view-logs.txt:130]:
```suggestion
:admin-api-endpoint: `Project & Application IDs
```

File: source/activity/view-logs.txt [hunk 4]
```diff
@@ -106,25 +200,25 @@ endpoint with either or both of the ``start_date`` and ``end_date``
 fields:
 
 .. note:: Result Pagination
-   
+
    If the date range that you specify includes more than 100 log
    entries, you will need to :ref:`run paginated queries
    <logging-get-paginated-logs>` to access all of the entries.
```
Comment (cbullinger) [inline source/activity/view-logs.txt:206]: might just be me, but I get slightly annoyed by inline links taking you somewhere on the same page when I'm not expecting it (and can't hit back to return to where you were...). maybe revise to make it more clear you're linking to the `Get Paginated Logs` section on this same page?

**Review by cbullinger — APPROVED** [review]
a couple things but otherwise lgtm

File: source/activity.txt (hunk 1 above)
Comment (nlarew) [reply source/activity.txt:114]: Accidentally left it off! Fixed.

File: source/activity/view-logs.txt (hunk 2 above)
Comment (nlarew) [reply source/activity/view-logs.txt:17]: Yeah - rephrasing to be about "logs" and not "the Logs screen"

**Pull request merged**